M2/Macaulay2/d/ctype.d

```diff
@@ -52,9 +51,24 @@ export isspace    (c:char):bool := (chartype(c) & SPACE    ) != 0;
 export isnewline  (c:char):bool := (chartype(c) & NEWLINE  ) != 0;
 export isquote    (c:char):bool := (chartype(c) & QUOTE    ) != 0;
 
-export isvalidsymbol  (s:string):bool := (
-     if int(uchar(s.0)) == 226 && length(s) == 3 then return true; -- ugly unicode math symbol hack
+export ismathoperator(c1:int, c2:int):bool := (
```

Comment:
if you're going to discriminate further, then you should probably also reduce the range of `c1 == 226`, only a subset of which are actual mathematical symbols, cf https://en.wikipedia.org/wiki/Mathematical_operators_and_symbols_in_Unicode

Reply:
It's up to Doug but I would rather this PR be limited and uncomplicated. Your suggestion could happen in a future PR.

Reply:
actually, it's literally a one-line change, in line with what was done for `c1 == 194`